bin/terminus.php

```diff
@@ -34,15 +39,26 @@
 $output = new ConsoleOutput();
 $container = Robo::createDefaultContainer($input, $output, $application, $config);
 
+$container->share('request', Request::class);
+$container->inflector(RequestAwareInterface::class)
+    ->invokeMethod('setRequest', ['request']);
+
 $container->share('fileCache', FileCache::class);
+
 $container->share('session', Session::class)
     ->withArgument('fileCache');
 $container->inflector(SessionAwareInterface::class)
     ->invokeMethod('setSession', ['session']);
+
 $container->share('sites', Sites::class);
 $container->inflector(SiteAwareInterface::class)
     ->invokeMethod('setSites', ['sites']);
 
+// Add our models
+$container->add(User::class);
```

Comment:
We could add names for these like we're doing for 'session', 'request', etc. But they already have perfectly good names so I don't see why we'd bother. They're not swappable services.